templates/firefox-ios.xliff

```diff
@@ -1597,6 +1684,14 @@
         <source>New Tab Settings</source>
         <note>Title displayed in header of the setting panel.</note>
       </trans-unit>
+      <trans-unit id="Settings.OfferClipboardBar.Status">
+        <source>When Opening Firefox</source>
+        <note>Status text of setting to enable the Go to Copied URL feature - https://bug1223660.bmoattachments.org/attachment.cgi?id=8898349</note>
```

Comment:
Flod mentioned to add a note here: "E.g.: Description displayed under the ”Offer to open copied link” option."